.circleci/config.yml

```diff
@@ -7,38 +7,63 @@ jobs:
 
     working_directory: ~/repo/
 
+    environment:
+      # Note: TEMP_FILE is hardcoded for save_cache and restore_cache. See dicussion on
+      # https://discuss.circleci.com/t/cannot-use-circle-yml-environment-variables-in-cache-keys/10994/20
+      TEMP_FILE: temp_requirements_summary.txt
```

Comment:
Something like `SUMMARY_REQUIREMENTS_FILE` or `SUM_REQ_FILE`, or `ALL_REQS` would be more meaningful than a generic `TEMP_FILE`...

Reply:
done. Chose `SUM_REQ_FILE`